elemica-suggest.coffee

```diff
@@ -180,6 +185,7 @@ elemicaSuggest 0.8.2-SNAPSHOT
             event.preventDefault()
           else if event.keyCode == TAB && isSelectingSuggestion()
             selectHighlighted(this)
+            removeSuggestions(this)
```

Comment:
I think it's pretty safe to say if we've made a selection we want the suggestions removed from the page, to be honest. I say let's move it there. If it becomes a problem we can always move it back.